src/services/options_init.ts

```diff
@@ -174,7 +174,38 @@ const defaultOptions: DefaultOption[] = [
 
     // Share settings
     { name: "redirectBareDomain", value: "false", isSynced: true },
-    { name: "showLoginInShareTheme", value: "false", isSynced: true }
+    { name: "showLoginInShareTheme", value: "false", isSynced: true },
+
+    // AI Options
+    { name: "aiEnabled", value: "false", isSynced: true },
+    { name: "openaiApiKey", value: "", isSynced: false },
+    { name: "openaiDefaultModel", value: "gpt-4o", isSynced: true },
+    { name: "openaiEmbeddingModel", value: "text-embedding-3-small", isSynced: true },
+    { name: "openaiBaseUrl", value: "https://api.openai.com/v1", isSynced: true },
+    { name: "anthropicApiKey", value: "", isSynced: false },
+    { name: "anthropicDefaultModel", value: "claude-3-opus-20240229", isSynced: true },
+    { name: "voyageEmbeddingModel", value: "voyage-2", isSynced: true },
+    { name: "voyageApiKey", value: "", isSynced: false },
+    { name: "anthropicBaseUrl", value: "https://api.anthropic.com/v1", isSynced: true },
+    { name: "ollamaEnabled", value: "false", isSynced: true },
+    { name: "ollamaDefaultModel", value: "llama3", isSynced: true },
+    { name: "ollamaBaseUrl", value: "", isSynced: true },
+    { name: "ollamaEmbeddingModel", value: "nomic-embed-text", isSynced: true },
+    { name: "embeddingAutoUpdateEnabled", value: "true", isSynced: true },
+
+    // Adding missing AI options
```

Comment:
bit of a nagging comment (sorry! :-D):
I don't think the comment is needed here, as the AI options are not missing anymore now ;-)